.gitmodules

```diff
@@ -70,3 +70,6 @@
 [submodule "src/drivers/pfsoc_crypto"]
 	path = src/drivers/pfsoc_crypto
 	url = ../pfsoc_crypto.git
+[submodule "platforms/pf_crypto"]
```

Comment:
This path is maybe not ideal, should be somewhere deeper under platforms; nuttx & mpfs specific.

Reply:
path changed to platforms/nuttxNuttX/extern/pf_crypto